README.md

```diff
@@ -79,6 +75,33 @@ Call **mx:nodes()** to get the list of mx cluster nodes.
 ['mxnode01@127.0.0.1','mxnode02@127.0.0.1']
 ```
 
+## Mnesia custom directory (optional)
+
+Create dir, for example, **/usr/local/var/lib/mx/mnesia/** with right permissions.
+
+Set `mnesia_base_dir` to that path in `sys.config`:
```

Comment:
Set option ... with this __directory__  in ... (you have to use the same term as you defined before)